### ISSUE TITLE
Handling large images - Weather Stories

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#2033]
- [x] Bump up upload limit to 3MB
- [x] Configure Drupal to overwrite (not append or add to) weather story images https://github.com/weather-gov/weather.gov/pull/2091
- [ ] Start tracking accumulated weather story image sizes on S3

**Comment by colinmurphy01** [comment]
@jamestranovich-noaa let's chat about this, if this is something we should consider for next sprint in order to get weather stories on our site consistantly.

**Comment by jamestranovich-noaa** [comment]
I proposed the following changes to the team:

- Bump up upload limit to 3MB
- Configure Drupal to overwrite (not append or add to) weather story images
- Start tracking accumulated weather story image on S3
- (Separate ticket) Figure out how to sync or share DSS builder uploads across all environments

Still waiting for input, so I will update this ticket as soon as we decide on what to do here!

**Comment by colinmurphy01** [comment]
@jamestranovich-noaa I think we agreed on a path forward for this, but let @coreypieper @shadkeene and @kmranjo know if you are still waiting on product to weigh in.

**Comment by jamestranovich-noaa** [comment]
Work has already started on this ticket so please let me know if we decide otherwise :sweat_smile: I'm currently working on `Configure Drupal to overwrite (not append or add to) weather story images` as we speak.

**Comment by jamestranovich-noaa** [comment]
as for monitoring s3 bucket sizes, we can't do this currently. the best we can do is to manually look at the s3 bucket size from time to time. 

as per the cloud.gov support team, though, it may be possible to ingest the cloudwatch metric for the bucket size into their new logging system based on opensearch. but tbd on that. for now, it's separated out: https://github.com/weather-gov/weather.gov/issues/2102

**Comment by jamestranovich-noaa** [comment]
We're good to go here. Closing.
